packages/data/tests/unit/serializers/bard-test.js

```diff
@@ -0,0 +1,638 @@
+import { module, test } from 'qunit';
```

Comment:
should this file be under `packages/data/tests/unit/serializers/metadata/bard-test.js` instead?

Reply:
Yes, I moved this in my next PR: https://github.com/yahoo/navi/pull/933/files#diff-7dbc9ec7e182639bf180f0b38ddae3ea :)